.build/install.ps1

```diff
@@ -1,17 +1,31 @@
+#!/usr/bin/pwsh
+
+###################
+## Setup PostGIS ##
 ###################
-### Setup PostgreSQL
-####################
 
-Write-Host Enabling PostGIS...
-If (!(Test-Path $env:POSTGIS_EXE)) {
-  Write-Host Downloading PostGIS...
-  (New-Object Net.WebClient).DownloadFile("http://download.osgeo.org/postgis/windows/pg10/$env:POSTGIS_EXE", "$env:POSTGIS_EXE")
+If ($isWindows) {
```

Comment:
So again, what I proposed previously is take out all the Windows-specific setup code into a Windows-specific ps1 file which you would execute in .appveyor.yml only on Windows. This would remove all the platform conditions inside Powershell scripts, allowing each script file to be either Linux, or Windows, or both.

Reply:
Hmmm... Okay, I misunderstood. This will introduce some (minimal) branching in the `.appveyor.yml`, but it does sound cleaner to keep platform-specific stuff separate.